mxcube3/routes/SampleCentring.py

```diff
@@ -264,8 +294,16 @@ def update_shapes():
         # Shape does not have any refs, create a new Centered position
         if not refs:
             x, y = shape_data["screen_coord"]
-            mpos = mxcube.diffractometer.get_centred_point_from_coord(x, y)
-            shape = mxcube.shapes.add_shape_from_mpos([mpos], (x, y), t)
+            mpos = mxcube.diffractometer.get_centred_point_from_coord(x, y, return_by_names=True)
+            pos.append(mpos)
+            # MD3 needs the center of the grid instead of the top left corner
```

Comment:
Okey we can keep the center, just make sure that we always have the top left corner and center for all grids independent of diffractometer. We can also remove the MD3 specific comment.